docs/concepts-broker-load-balancing-quick-start.md

```diff
@@ -0,0 +1,594 @@
+
+**Input**
+
+```bash
+docker exec -it broker-2 bash
+```
+
+**Output**
+
+If there is no output, the operation is successful.
+
+5.3 Verify whether the bundles are split after 5 minutes. It takes a little time for Pulsar to split bundles.
+
+:::tip
+
+- The automatic bundle splitting is **enabled by default**. To disable it, update the following configurations to false in the broker.conf file. 
+  
+    - [loadBalancerAutoBundleSplitEnabled](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1278)
+  
+    - [loadBalancerAutoUnloadSplitBundlesEnabled](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1281C4-L1281C4) 
+
+- The default bundle splitting algorithm is `range_equally_divide`. You can change it to other [bundle splitting algorithms](./concepts-broker-load-balancing-concepts.md#bundle-splitting-algorithms) by updating [defaultNamespaceBundleSplitAlgorithm](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1321C18-L1321C18) in the broker.conf file. 
+
+- For bundle splitting thresholds, you can set more configurations in the broker.conf file. Any existing bundle that exceeds any of the thresholds is a candidate to be split. 
+:::
+
+**Input**
+
+```bash
+bin/pulsar-admin namespaces bundles my-tenant/my-namespace
+```
+
+**Output**
+
+The output shows that the bundle has been split into 3 ranges.
+
+```bash
+{
+  "boundaries" : [ "0x00000000", "0x3fffffff", "0x7fffffff", "0xffffffff" ],
+  "numBundles" : 3
+}
+```
+
+:::note
+
+The number of bundles may vary depending on the time. For example, if you wait longer than 5 minutes, the number of bundles may increase; otherwise, the number of bundles may decrease. If the number of bundles is equal to or greater than 2, it means that the bundle has been split.
+:::
+
+### Step 6. Unload bundles
+
+You can verify if a bundle has been unloaded using the metric [pulsar_lb_unload_bundle_total](./reference-metrics.md#load-balancing). This method is only supported in the automatic way.
+
+:::tip
+
+- The automatic bundle unloading is **enabled by default**. To disable it, update [loadBalancerSheddingEnabled](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1259C14-L1259C14) to false in the broker.conf file. 
+
+- The **default** bundle unloading strategy is TransferShedder (for the extensible broker load balancer) or ThresholdShedder (for the modular broker load balancer). You can change it to other [bundle unloading strategies](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies) by updating [loadBalancerLoadSheddingStrategy](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1324C52-L1324C52) in the broker.conf file. 
+
+- For bundle unloading conditions, you can set more configurations in the broker.conf file. 
```

Comment:
For reviewers: I'll add the configuration link once that link is available (the configuration doc PR is WIP).